services/rds/src/main/java/org/finra/gatekeeper/services/accessrequest/delegates/GrantAccessServiceTask.java

```diff
@@ -70,6 +72,7 @@ public void execute(DelegateExecution execution) throws Exception {
 
         AccessRequest accessRequest = (AccessRequest) execution.getVariable("accessRequest");
         logger.info("Granting Access to " + accessRequest);
+        RequestEventLogger.logEventToJson(EventType.AccessGranted, accessRequest);
```

Comment:
We should move this to the bottom of this method. If approving the request craps out, then this will get incorrectly reflected.